### PR TITLE
adding ppc64le support and making build script changes for install li…

### PR DESCRIPTION
**Pull request by amukherjee28**
adding ppc64le support and making build script changes for install libnuma support.

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Review by xhochy — CHANGES_REQUESTED** [review]
Please read up on CDTs, this is what you need here. We already have a CDT for `numa-devel` for CentOS 6, we can also build one for Cent OS 7 if that is needed here.

**Comment by amukherjee28** [comment]
@xhochy 

Could you please explain a bit as to where we might need the change, Where can I check the CDT and how can I  apply it here,
It would really help a lot.

Thanks.

**Comment by xhochy** [comment]
Have a look at https://conda-forge.org/docs/maintainer/knowledge_base.html#core-dependency-tree-packages-cdts

**Comment by amukherjee28** [comment]
@xhochy 
Can You review the PR now that all checks for linux_64, windows, mac and ppc64le has passed.

**Comment by xhochy** [comment]
This still not using CDTs and all changes would get removed automatically by calling "@conda-forge-admin please rerender". Please don't touch the autogenerated scripts.

**Comment by amukherjee28** [comment]
@xhochy 
understood my changes in .scripts/build_steps.py shall be removed.

Oki I will continue to work in this PR to see how to add the required dependency via recipe.yaml

**Comment by amukherjee28** [comment]
@xhochy 

I was working on this PR and trying to add the CDT for libnuma.

Adding ``` {{ cdt('numactl-devel') }}``` in the build section of meta.yaml does install the package ```numactl-devel-cos7-ppc64le``` but the library goes to the different directory ```opt/conda/pkgs/numactl-libs-cos7-ppc64le-2.0.12-h9b0a68f_1105/powerpc64le-conda-linux-gnu/sysroot/usr/lib64/libnuma.so.1```

However while the dependency still get missing as the python library tries to resolve the dependency from /lib64 folder.

Any insight in case you can provide on how we can take care of this kind of situation.

**Comment by amukherjee28** [comment]
@xhochy can you please review this PR once.
The mac jobs are failing because of the missing libstdc++ library. However we are able to make progress with the ppc64le jobs.

I have introduced the cdt numactl-devel along with yum_requirements.txt to have to library installed only in power systems.

With this all of the ppc64le jobs have passed after rerender.

Can you please now a overlook into the PR and see if this case me merged.

Thanks.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ requirements:
     - python
     - setuptools
     - libxml2
+    - {{ cdt('numactl-devel') }}  # [ppc64le]
```
Comment (xhochy) [inline recipe/meta.yaml:30]: CDTs should be in `build`, see https://conda-forge.org/docs/maintainer/knowledge_base.html#core-dependency-tree-packages-cdts

**Review by xhochy — COMMENTED** [review]
The OSX failure is weird in the sense that it should have never worked before. Can you skip it?

**Comment by amukherjee28** [comment]
@xhochy Even I am not able to figure why suddenly I am seeing the OSX failure. This is due to the missing library libstdc++.
I am not sure how I skip OSX build as linux, windows and Mac build happens by default.

With respect to CDT change I will put the changes under build section.

File: recipe/meta.yaml (hunk 1 above)
Comment (amukherjee28) [reply recipe/meta.yaml]: Fixed this change in the latest commit

**Comment by amukherjee28** [comment]
@conda-forge-admin please rerender

**Comment by amukherjee28** [comment]
@xhochy we have fixed the OSX failure. Can you please let me know if there is a way to re-run the checks without making any more commits.

Thanks

**Review by xhochy — APPROVED** [review]
Thanks for keeping the patience and fixing this the right way!

**Comment by amukherjee28** [comment]
Thanks a lot @xhochy for your help and guidance.

Thanks for merging the PR.

**Pull request merged**